wsproto/frame_protocol.py

```diff
@@ -578,14 +575,14 @@ def ping(self, payload: bytes = b"") -> bytes:
     def pong(self, payload: bytes = b"") -> bytes:
         return self._serialize_frame(Opcode.PONG, payload)
 
-    def send_data(  # type: ignore
-        self, payload: AnyStr = b"", fin: bool = True
+    def send_data(
+        self, payload: Union[bytes, bytearray, str] = b"", fin: bool = True
     ) -> bytes:
-        if isinstance(payload, (bytes, bytearray, memoryview)):
```

Comment:
Maybe we should support `memoryview` here?

Reply:
I'll bring it back. As mentioned in the PR description, I think the types here need some sorting out, but better do it separately from these changes.